pkg/cmd/api/api.go

```diff
@@ -176,6 +189,7 @@ func NewCmdApi(f *cmdutil.Factory, runF func(*ApiOptions) error) *cobra.Command
 	cmd.Flags().BoolVar(&opts.Paginate, "paginate", false, "Make additional HTTP requests to fetch all pages of results")
 	cmd.Flags().StringVar(&opts.RequestInputFile, "input", "", "The `file` to use as body for the HTTP request")
 	cmd.Flags().BoolVar(&opts.Silent, "silent", false, "Do not print the response body")
+	cmd.Flags().StringVar(&cacheDuration, "cache", "", "Cache the response for the specified duration")
```

Comment:
As a user I'd be confused what value to put here -- it's not intuitive that something like "1m" would work without an example. How about a default like `"1m"` to hint at what kind of string we accept here?

Reply:
How about using `cmd.Flags().DurationVar`?

Reply:
@gagliardetto Oh I didn't know about the dedicated method! Updated; thanks!